package.json

```diff
@@ -19,14 +19,20 @@
     "url": "https://github.com/ember-cli/ember-router-generator.git"
   },
   "dependencies": {
+    "@babel/parser": "^7.4.5",
+    "@babel/traverse": "^7.4.5",
     "recast": "^0.18.1"
   },
   "devDependencies": {
     "assertion-error": "^1.1.0",
     "chai": "^4.2.0",
-    "esprima": "^2.7.2",
+    "eslint": "^6.0.1",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-mocha": "^5.3.0",
+    "eslint-plugin-node": "^9.1.0",
```

Comment:
if it's not too much work, could you extract the JSHint->ESLint change to a separate PR? that would make the parser changes here significantly easier to review 🙏

Reply:
I tried to separate it completely but it was non-trivial because some of the code in this PR was incompatible with the old version of jshint. I tried to make the code changes as minimal as possible. If it makes it easier, almost all parser-related changes are in the code, not configuration files (other than the new Babel deps in `package.json`).

Reply:
sounds like an ordering problem to me. if switching to ESLint is necessary for the code in this PR then why not switch to ESLint first, before modifying any code? I assume the ESLint change does not depend on the new parser code in this PR.